### DISCUSSION TITLE
Brazilian mask

### DISCUSSION THREAD

**Discussion by adrielschmitz** [typesnippet/react-phone-hooks#50] [General]
As shown in the link below, in 2017 Brazilian telephone companies had to add the digit 9 before the number, so the number that was previously (12) 1234 1234 became (12) 91234 1234. Above all, this change is not mandatory for all numbers, so here in Brazil we have to deal with the option of the number being in the first or second format. Today, in the project, in the country Brazil (br), both formats are already accepted, however, the applied mask is as (12) 1234 12345, however, there is a kind of "gentlemen's agreement" between the software here and the standard is: (12) 12345-1234.
With that in mind, I would like to suggest changing the mask so that instead of `"+55 (..) .... ....."`, it is `"+55 (..) .....-...."`
https://www.gov.br/anatel/pt-br/regulado/numeracao/nono-digito

**Comment by ArtyomVancyan** [comment]
I have updated and just published the new release. You can upgrade the `react-phone-hooks` version you're using and enjoy.
